.github/workflows/ebpf_linux.yml

```diff
@@ -78,6 +77,33 @@ jobs:
     - name: Build
       run: |
         cmake --build ${{github.workspace}}/build --config Release -- -j $(nproc)
+        cp libbpf/src/libbpf.so ${{github.workspace}}/build/bin
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: bpf_performance
+        path: ${{github.workspace}}/build/bin
+
+  test:
+    needs: [build]
+    runs-on: 
+    - self-hosted
+    - os-ubuntu-20.04
+    - x64
+    - azure
+
+    steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: bpf_performance
+        path: ${{github.workspace}}/bpf_performance
+
+    - name: Setup
```

Comment:
Looks like this didn't help with the error:
```
Run ldd ./bpf_performance_runner
./bpf_performance_runner: /lib/x8[6](https://github.com/microsoft/netperf/actions/runs/8754533047/job/24026563428?pr=164#step:6:7)_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./bpf_performance_runner)
./bpf_performance_runner: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./bpf_performance_runner)
./bpf_performance_runner: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by ./bpf_performance_runner)
	linux-vdso.so.1 (0x0000[7](https://github.com/microsoft/netperf/actions/runs/8754533047/job/24026563428?pr=164#step:6:8)ffee9d8e000)
	libbpf.so.0 => not found
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f4ae64e2000)
	libgcc_s.so.1 => /lib/x[8](https://github.com/microsoft/netperf/actions/runs/8754533047/job/24026563428?pr=164#step:6:9)6_64-linux-gnu/libgcc_s.so.1 (0x00007f4ae64c7000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f4ae62d5000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f4ae6186000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f4ae674f000)
sudo: ./bpf_performance_runner: command not found
```

Reply:
Copilot suggestion the following:

Install the Latest libstdc++6: This will update the GLIBCXX version on your system.
```bash
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt-get update
sudo apt-get install --only-upgrade libstdc++6
```

Check for Available Versions: After installation, check if the required versions are now available.
```bash
strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
```

Upgrade GLIBC: If the required GLIBC versions are still not available, you may need to upgrade your GLIBC. The following commands will help you upgrade your system which should include the latest GLIBC.
```bash
sudo apt-get update
sudo apt-get upgrade
sudo apt-get dist-upgrade
```

Please note that adding the PPA and upgrading your system libraries can have significant effects on your system, so proceed with caution. [Ensure that you understand the changes that will be made to your system and consider backing up important data before proceeding](https://askubuntu.com/questions/575505/glibcxx-3-4-20-not-found-how-to-fix-this-error/764572). [1](https://askubuntu.com/questions/575505/glibcxx-3-4-20-not-found-how-to-fix-this-error/764572) [2](https://stackoverflow.com/questions/76974555/glibcxx-3-4-32-not-found-error-at-runtime-gcc-13-2-0) [3](https://hatchjs.com/glibc_2-28-not-found-ubuntu/)